### PR TITLE
multi: fix some function names in interface comment

### PR DESCRIPTION
**Pull request by chuangjinglu**
## Change Description

fix some function names in interface comment

## Steps to Test
Steps for reviewers to follow to test the change.

### Testing
- [ ] Your PR passes all CI checks.
- [ ] Tests covering the positive and negative (error paths) are included.
- [ ] Bug fixes contain tests triggering the bug to prevent regressions.

### Code Style and Documentation
- [ ] The change obeys the [Code Documentation and Commenting](https://github.com/lightningnetwork/lnd/blob/master/docs/code_contribution_guidelines.md#CodeDocumentation) guidelines, and lines wrap at 80.
- [ ] Commits follow the [Ideal Git Commit Structure](https://github.com/lightningnetwork/lnd/blob/master/docs/code_contribution_guidelines.md#IdealGitCommitStructure).
- [ ] Any new logging statements use an appropriate subsystem and logging level.
- [ ] Any new lncli commands have appropriate tags in the comments for the rpc in the proto file.
- [ ] [There is a change description in the release notes](https://github.com/lightningnetwork/lnd/tree/master/docs/release-notes), or `[skip ci]` in the commit message for small changes.

📝 Please see our [Contribution Guidelines](https://github.com/lightningnetwork/lnd/blob/master/docs/code_contribution_guidelines.md) for further guidance.

### CODE REVIEW

File: sweep/walletsweep.go [hunk 1]
```diff
@@ -152,7 +152,7 @@ func (f FeeEstimateInfo) Estimate(estimator chainfee.Estimator,
 // UtxoSource is an interface that allows a caller to access a source of UTXOs
 // to use when crafting sweep transactions.
 type UtxoSource interface {
-	// ListUnspentWitness returns all UTXOs from the default wallet account
+	// ListUnspentWitnessFromDefaultAccount returns all UTXOs from the default wallet account
```
Comment (ziggie1984) [inline sweep/walletsweep.go:155]: wrap lines to 80 columns otherwise the linter will complain.

**Review by ziggie1984 — CHANGES_REQUESTED** [review]
Thanks for the fix, can you make sure the linter passes.

File: input/input.go [hunk 2]
```diff
@@ -19,7 +19,7 @@ var EmptyOutPoint wire.OutPoint
 // construct a valid input within a sweeping transaction to sweep this
 // lingering UTXO.
 type Input interface {
-	// Outpoint returns the reference to the output being spent, used to
+	// OutPoint returns the reference to the output being spent, used to
```
Comment (ziggie1984) [inline input/input.go:22]: Also in the commit msg, use `multi` because you are fixing typos in multiple packages

File: sweep/walletsweep.go (hunk 1 above)
Comment (chuangjinglu) [reply sweep/walletsweep.go]: Thanks for your guidance, I resized the length of each line.

File: input/input.go (hunk 2 above)
Comment (chuangjinglu) [reply input/input.go:22]: Modified

**Review by yyforyongyu — APPROVED** [review]

**Review by ziggie1984 — APPROVED** [review]

**Pull request merged**